src/frontend/komponenter/Personlinje/Personlinje.tsx

```diff
@@ -89,7 +91,12 @@ export function Personlinje({ bruker, fagsak }: Props) {
                     />
                     <HStack align={'center'} gap={'3 4'}>
                         <BodyShort as={'span'} weight={'semibold'}>
-                            {fagsakeier.navn} ({fagsakeier.alder} år)
+                            {fagsakeier.navn} ({fagsakeier.alder} år){' '}
```

Comment:
Usikker på om du trenger `{' '}` 

```suggestion
                            {fagsakeier.navn} ({fagsakeier.alder} år)
```